pyproject.toml

```diff
@@ -19,17 +19,17 @@ requires-python = ">=3.8"
 dynamic = ["version"]
 
 dependencies = [
-    "qiskit==0.36.0",
-    "pytket==1.2.2",
-    "pytket-qiskit==0.25.0",
+    "qiskit~=0.39.2",
+    "pytket~=1.8.1",
+    "pytket-qiskit~=0.30.0",
     "pandas==1.3.5",
     "flask==2.2.2",
     "networkx==2.8.6",
     "pytest==7.1.3",
-    "qiskit_finance==0.3.4",
-    "qiskit_machine-learning==0.4.0",
-    "qiskit-nature[pyscf]==0.3.1",
-    "qiskit_optimization==0.3.2",
+    "qiskit_finance~=0.3.4",
+    "qiskit_machine-learning~=0.5.0",
+    "qiskit-nature[pyscf]~=0.5.1",
+    "qiskit_optimization~=0.4.0",
```

Comment:
In the future, we might have to be even more lenient on the Qiskit/TKET versions here in order to avoid the package frequently breaking. No action item as of now. Feel free to resolve.